tests/CMakeLists.txt

```diff
@@ -196,3 +196,23 @@ add_test(
 )
 set_tests_properties("test-generate_contributors" PROPERTIES LABELS quick)
 endif()
+
+
+if (SSG_PRODUCT_RHEL8 AND PYTHON3_EXECUTABLE)
+add_test(
+        NAME "test-create_scap_delta_tailoring"
+        COMMAND env "PYTHONPATH=$ENV{PYTHONPATH}" "${PYTHON_EXECUTABLE}" "${CMAKE_SOURCE_DIR}/utils/create_scap_delta_tailoring.py" --root "${CMAKE_SOURCE_DIR}" --json "${CMAKE_BINARY_DIR}/rule_dirs.json" --build-config-yaml "${CMAKE_BINARY_DIR}/build_config.yml" --reference "stigid"  --output "${CMAKE_BINARY_DIR}/rhel8_stig_tailoring.xml" --product rhel8 --manual  "${RHEL8_DISA_STIG_REF}" -B "${CMAKE_BINARY_DIR}"
+)
+set_tests_properties("test-create_scap_delta_tailoring" PROPERTIES LABELS quick)
```

Comment:
I would add the --dry-run here as well and hopefully it will make the test faster. It takes about 10 seconds so I wouldn't label it as "quick".